practices/practice-two/src/styles/sections/better-search-template.css

```diff
@@ -0,0 +1,109 @@
+.template-wrapper {
+  background-color: var(--sixth-color);
+}
+
+.template-item {
+  display: grid;
+  gap: 30px;
+}
+
+.template-image {
+  background-color: transparent;
+  position: relative;
+}
+
+.image {
+  width: 508px;
```

Comment:
How about when image is large than 508px?

Reply:
I set 508px as the default on the design, when on mobile it is 100%